nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/change-color-dialog/change-color-dialog.component.html

```diff
@@ -0,0 +1,65 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<h2 mat-dialog-title>Change Color</h2>
+<form [formGroup]="changeColorForm">
+    <mat-dialog-content>
+        <div class="flex flex-col">
+            <mat-form-field>
+                <mat-label>Color</mat-label>
+                <input matInput type="color" formControlName="color" (change)="colorChanged()" />
+            </mat-form-field>
+            <mat-checkbox color="primary" [checked]="noColor" formControlName="noColor" (change)="noColorChanged()"
+                >Use default color</mat-checkbox
+            >
+        </div>
+        <div class="pt-4 preview">
+            <div class="value">Preview</div>
+            @if (type === ComponentType.Processor) {
+                <div class="processor border drop-shadow-lg" [style.border-color]="color">
+                    <div class="component-context flex flex-col">
+                        <div class="flex gap-x-1 items-center">
+                            <div class="context-logo flex flex-col items-center" [style.background-color]="color">
+                                <i
```

Comment:
These paddings seem a bit off in purple mode. Likely because you were attempting to reuse the context-component styles.